docs/_static/css/custom.css

```diff
@@ -14,3 +14,12 @@ dl.class em:not([class]):last-of-type::after {
 dl.class > dt:first-of-type {
     display: block !important;
 }
+
+button.copybtn {
+    height:25px;
+    width:25px;
+    opacity: 0.5;
```

Comment:
This way the button is always visible but faded (50% visibility) even when the mouse is not hovered over the code block. I can change it to be only visible when hovered over, background etc. if you prefer it differently.